src/Tribe/Editor/Configuration.php

```diff
@@ -24,8 +64,12 @@ public function hook() {
 	 * @return array
 	 */
 	public function editor_config( $editor_config ) {
-		$tec = empty( $editor_config['events'] ) ? array() : $editor_config['events'];
+		$tec = empty( $editor_config['events'] ) ? [] : $editor_config['events'];
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 21 spaces but found 1 space

<sub>hash: 05f1ae9c2ad8e9496d259bd5b1efacab</sub>